### ISSUE TITLE
Document location of prebuilt static libraries

### ISSUE THREAD

**Issue by lugehorsam** [spotify/NFHTTP#33]
The docs mention the existence of prebuilt libraries but not where to find them.

My use case is linking to this project via Bazel rather than using CMake.

**Comment by drubinstein** [comment]
The libraries we use can be found in the `libraries` directory. 
https://github.com/spotify/NFHTTP/tree/master/libraries

**Comment by lugehorsam** [comment]
Ah, I thought from this line `NFHTTP is a Cmake project, while you are free to download the prebuilt static libraries...(etc)` that NFHTTP itself came in a prebuilt form.

Not needed anymore for me since the Bazel idiom is to build from source anyway.

**Comment by nu8** [comment]
I agree that line is pretty misleading, as prebuilt means... already built. It doesnt mean "build it yourself".
